Triangulation_2/include/CGAL/Triangulation_2/internal/Polyline_constraint_hierarchy_2.h

```diff
@@ -171,8 +162,11 @@ class Polyline_constraint_hierarchy_2
   typedef typename Context_list::iterator Context_iterator;
 
   typedef std::set<Constraint_id>           Constraint_set;
-  typedef std::map<Edge, Context_list*,
-                   Pair_compare>            Sc_to_c_map;
+#if CGAL_USE_BOOST_UNORDERED
```

Comment:
Would  it make sense to have a template alias to automatically choose the fastest one?

Reply:
For the moment, all those changes are only in my CDT branch (named `lrineau/Triangulation_3-CDT_3-lrineau`). I still have to backport useful patches to CGAL `master`, via a pull-request.

I agree an alias would be nice to have.